src/test/java/seedu/address/logic/commands/AnakinCommandTestUtil.java

```diff
@@ -63,6 +70,17 @@
 
     public static final String PREAMBLE_WHITESPACE = "\t  \r  \n";
     public static final String PREAMBLE_NON_EMPTY = "NonEmptyPreamble";
+    public static final EditDeckDescriptor DESC_AMY;
+    public static final EditDeckDescriptor DESC_BOB;
+
+    static {
+        DESC_AMY = new EditDeckDescriptorBuilder().withName(VALID_NAME_DECK_A)
```

Comment:
Are you keeping AMY and BOB?

Reply:
I deleted the Amy and Bob PersonDescriptors and changed them to DeckDescriptors